hackathon_site/dashboard/frontend/src/pages/TeamDetail/TeamDetail.tsx

```diff
@@ -31,24 +32,15 @@ const TeamDetail = ({ match }: RouteComponentProps<PageParams>) => {
                     </Typography>
                 </Grid>
 
-                <Grid item container direction="row" spacing={2}>
+                <Grid
+                    item
+                    container
+                    spacing={1}
```

Comment:
increase the spacing here (from my previous comment), unless there's something i'm not seeing correctly pls lmk

Reply:
ill try it instead of the margin and see!